extra/redisotel/redisotel.go

```diff
@@ -136,3 +142,10 @@ func WithAttributes(attrs ...attribute.KeyValue) Option {
 		cfg.attrs = append(cfg.attrs, attrs...)
 	})
 }
+
+// WithIgnoreRawCmd tells the tracing hook not to log raw redis commands.
+func WithIgnoreRawCmd() Option {
```

Comment:
Let's rename this to `WithDBStatement(true/false)` so it can be more easily enabled/disabled - https://uptrace.dev/blog/posts/go-functional-options-named-args.html#bool-options